modules/local/deepvariant/functions.nf

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
Haven't made any changes in the `functions.nf` file - is any customization required here?

Reply:
Nothing needs to be done here